manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
```

Comment:
Yeah, putting it in it's own $ssldir_manage parameter.

The example I could think of would be if they were already managing that directory in their profile in some way, you'd get a duplicate resource error, so upgrading the module would be a breaking change...

I'm generally more paranoid about this kind of stuff, and usually parameterise all changes, there's a few different opinions on this. Anyone else got thoughts?